### PR TITLE
Adding get command

### PR DESCRIPTION
**Pull request by jcscottiii**

### CODE REVIEW

File: config/common/mocks/BaseSchema.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+type BaseSchema struct {
+	mock.Mock
+}
+
+// Parse provides a mock function with given fields: data
+func (_m *BaseSchema) Parse(data []byte) error {
+	ret := _m.Called(data)
+
+	var r0 error
```
Comment (geramirez) [inline config/common/mocks/BaseSchema.go]: any specific reason for calling the error r0?

Comment (jcscottiii) [inline config/common/mocks/BaseSchema.go]: part of the mockery tool autogen

File: config/versions/1.0.0/tools/resourceGetter.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// GetRemoteResources is the implementation that uses VCS to get remote resources.
+func (g VCSAndLocalFSGetter) GetRemoteResources(destination string, subfolder string, worker *common.ConfigWorker, entries []common.Entry) error {
+	tempResourcesDir, err := ioutil.TempDir("", "opencontrol-resources")
```
Comment (geramirez) [inline config/versions/1.0.0/tools/resourceGetter.go]: So this places all the files a temp dir while it's getting the new directories? Is the part that moves it into opencontrols implemented yet?

Comment (geramirez) [inline config/versions/1.0.0/tools/resourceGetter.go]: nvm

Comment (geramirez) [inline config/versions/1.0.0/tools/resourceGetter.go]: Actually, how do we stop (or will stop) duplicate dependencies from being downloaded? Maybe we could use something like [safeDict](https://github.com/18F/angrytock/tree/master/safeDict) so we can process dependencies concurrently.

**Comment by geramirez** [comment]
This is great :+1: and well tested.

**Comment by geramirez** [comment]
Would be this be ready for integration test?

**Pull request merged**